snapcraft/extensions/kde_neon_6.py

```diff
@@ -0,0 +1,272 @@
+            kf6_builtin=kf6_builtin,
+        )
+
+    @functools.cached_property
+    def ext_info(self) -> ExtensionInfo:
+        """Return the extension info cmake_args, provider, content, build_snaps."""
+        qt6_cmake = f"/snap/{self.kde_snaps.qt6_sdk_snap}/current;"
+        kf6_cmake = f"/snap/{self.kde_snaps.kf6_sdk_snap}/current"
+        cmake_args = "-DCMAKE_FIND_ROOT_PATH=" + qt6_cmake + kf6_cmake
+
+        return ExtensionInfo(cmake_args=cmake_args)
+
+    @overrides
+    def get_root_snippet(self) -> Dict[str, Any]:
+        platform_qt6_snap = self.kde_snaps.content_qt6
+        content_qt6_snap = self.kde_snaps.content_qt6 + "-all"
+        platform_kf6_snap = self.kde_snaps.content_kf6
+        content_kf6_snap = self.kde_snaps.content_kf6 + "-all"
+
+        return {
+            "assumes": ["snapd2.58.3"],  # for 'snapctl is-connected'
+            "compression": "lzo",
+            "plugs": {
+                "desktop": {"mount-host-font-cache": False},
+                "icon-themes": {
+                    "interface": "content",
+                    "target": "$SNAP/data-dir/icons",
+                    "default-provider": "gtk-common-themes",
+                },
+                "sound-themes": {
+                    "interface": "content",
+                    "target": "$SNAP/data-dir/sounds",
+                    "default-provider": "gtk-common-themes",
+                },
+                platform_qt6_snap: {
+                    "content": content_qt6_snap,
+                    "interface": "content",
+                    "default-provider": platform_qt6_snap,
+                    "target": "$SNAP/qt6",
+                },
+                platform_kf6_snap: {
+                    "content": content_kf6_snap,
+                    "interface": "content",
+                    "default-provider": platform_kf6_snap,
+                    "target": "$SNAP/kf6",
+                },
+            },
+            "environment": {"SNAP_DESKTOP_RUNTIME": "$SNAP/kf6"},
+            "hooks": {
+                "configure": {
+                    "plugs": ["desktop"],
+                    "command-chain": ["snap/command-chain/hooks-configure-desktop"],
+                }
+            },
+            "layout": {"/usr/share/X11": {"symlink": "$SNAP/kf6/usr/share/X11"}},
+        }
+
+    @overrides
+    def get_part_snippet(self, *, plugin_name: str) -> Dict[str, Any]:
+        qt6_sdk_snap = self.kde_snaps.qt6_sdk_snap
+        kf6_sdk_snap = self.kde_snaps.kf6_sdk_snap
+        cmake_args = self.ext_info.cmake_args
+
+        return {
+            "build-environment": [
+                {
+                    "PATH": prepend_to_env(
+                        "PATH",
+                        [
+                            f"/snap/{qt6_sdk_snap}/current/"
+                            f"usr/bin:/snap/{kf6_sdk_snap}/"
+                            f"current/usr/bin"
+                        ],
```

Comment:
```suggestion
                        [
                            f"/snap/{qt6_sdk_snap}/current/usr/bin",
                            f"/snap/{kf6_sdk_snap}/current/usr/bin",
                        ],
```

This looks more readable to me